### DISCUSSION TITLE
NPC True Tile

### DISCUSSION THREAD

**Discussion by tiptop103** [runelite/runelite#15121] [Q&A] [answered]
Could you add the ability to have NPC indicators show an NPC's full true tile when selected? It exists on other clients and is fully within Jagex's rules.

Moreover, I think reworking the entire NPC indicator menu would be a good idea, so you can selectively choose to highlight hull/tile/south-west/true tile for specific NPCs instead of having it do all.

Example of better NPC menu: https://imgur.com/a/PgqlGng

**Comment by RonnyTommy548** [comment]
I also would appreciate a feature to highlight the full true tile of monsters, instead of just the SW true tile (which was just added to RuneLite today). Currently, to achieve this you have to get the NPC ID plugin and Radius Markers plugin from the plugin hub. Monsters commonly have multiple different IDs, so you would have to get every ID and enter them into Radius Markers if you actually want to see the full true tile for a monster. For example, if you want to see the true tile of Nex, you would have to make 3 entries in radius markers for IDs 11278, 11280, 11281.

**Comment by Adam-** [comment]
The SW tile is true tile but it is 1x1 - so for clarification you wish to see true tile but for a >1x1 npc?

**Reply by RonnyTommy548** [reply]
Yes, the full true tile of the NPC. For example, verzik P3 would be a 7x7, so if you are trying to pog tank and you want to step under on a specific tile of verzik while she is dragging around the room, you don't have to use your imagination to extend the SW 1x1 true tile when you are on the NE side of verzik to know which tile truly is and is not under verzik

**Reply by Adam-** [reply]
Fair enough, will work on this today

**Reply by tiptop103** [reply]
Ronny has already described it, but yes, it's quite nice for tekton and he has different orientations in different instances of regular raids, so having it for the entire npc highlight would be very nice.

**Comment by Adam-** [comment]
Is the sw-tile highlight useful if there is a separate true tile setting for >1x1 npcs?

**Reply by tiptop103** [reply]
It can still be useful in fight caves/inferno so you can instantly recognise where the south west tile is to allow chinchompa/barrage clumps, so I would keep it personally

**Comment by Adam-** [comment, marked as answer]
This is now added
